Code/Framework/AzToolsFramework/Tests/ManipulatorViewTests.cpp

```diff
@@ -51,12 +54,9 @@ namespace UnitTest
         ///////////////////////////////////////////////////////////////////////////////////////////////////////////////
         // Given
         const AZ::Transform orientation =
-            AZ::Transform::CreateFromQuaternion(
-                AZ::Quaternion::CreateFromAxisAngle(
-                    AZ::Vector3::CreateAxisX(), AZ::DegToRad(-90.0f)));
+            AZ::Transform::CreateFromQuaternion(AZ::Quaternion::CreateFromAxisAngle(AZ::Vector3::CreateAxisX(), AZ::DegToRad(-90.0f)));
```

Comment:
not a big deal, but I think you could also get the same transform more succinctly with

AZ::Transform::CreateRotationX(AZ::DegToRad(-90.0f))

Reply:
Oh yeah that's a good shout, I can update that 👍

Reply:
Now fixed